app/views/search/single_facility.html.erb

```diff
@@ -0,0 +1,88 @@
+<% provide(:title, "Single Facility") %>
+<div id="content" class="interior">
+  <a id="content" href="#content" class="sr-only">Content starts here.</a>
+
+  <div class="splash">
+    <div class="row">
+      <div class="small-12 columns">
+        <ul class="breadcrumbs" role="menubar" aria-label="Primary">
+          <li class="active"><h2>Service And Facility Locator</h2></li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</div> 
+
+<div class="section one">
+    <div class="row">
+        <div class="small-12 column">
+            <h2><%= @facility.Location_Descriptive_Name_Common_Name %></h2>
+        </div>
+    </div>
+    <div class="row">
+  <div class="small-4 columns">
+<!--
```

Comment:
Nitpick: Can delete comments. 
Tip: Rails has a web console that you can bring up in dev mode that lets you issue commands to reveal this. Or if you're using RubyMine, you can set a breakpoint in the ERB.